package.json

```diff
@@ -30,7 +30,7 @@
     "test:watch": "jest --watch --verbose false",
     "test:update": "jest --updateSnapshot",
     "test:debug": "node --inspect-brk $(npm bin)/jest --runInBand --watch --verbose false",
-    "test:e2e": "CYPRESS_baseUrl=http://localhost:9999 cypress run --browser chrome --reporter junit --reporter-options 'mochaFile=junit-results/test-output-[hash].xml'",
+    "test:e2e:kind": "CYPRESS_inkind=true CYPRESS_bucket=defbuck CYPRESS_baseUrl=https://twodotoh.a.influxcloud.dev.local CYPRESS_password=password cypress open --browser chrome --config chromeWebSecurity=false",
```

Comment:
this is pretty gross, why not run it with a cypress.config.json (which already cover bucket and password), and there's nothing special about it being in kind right? just pointing to a new ingress, which we're gonna have to bake / share in our dev images anyways (like how INGRESS_HOST is currently used)?

not blockers, just opportunities to clean things up

Reply:
I didn't mean to delete `test:e2e` command, I'll remove `CYPRESS_inkind`